jetty-http/src/main/java/org/eclipse/jetty/http/HttpCookie.java

```diff
@@ -29,6 +29,21 @@
     private static final String __COOKIE_DELIM = "\",;\\ \t";
     private static final String __01Jan1970_COOKIE = DateGenerator.formatCookieDate(0).trim();
 
+    public enum SameSite
+    {
+        EXCLUDED("Excluded"), NONE("None"), STRICT("Strict"), LAX("Lax");
```

Comment:
I don't like having `Excluded` as a fake value that is not defined in the RFC (which is onlyr really https://tools.ietf.org/html/draft-west-first-party-cookies-07 -right?)

I prefer to just have the `_sameSite=null;` to mean excluded.

I also note that the bnf in the spec doesn't really define `None` either: https://tools.ietf.org/html/draft-west-first-party-cookies-07#section-3.1  Oh well - it is in the text!

Reply:
I've made this change.